### DISCUSSION TITLE
Permanently set "Custom Battery Capacity (kWh) when new"?

### DISCUSSION THREAD

**Discussion by dlover25** [teslamate-org/teslamate#3820] [Q&A]
Is there a way to permanently set the "Custom Battery Capacity (kWh) when new" field on the Battery Health Screen? I have some wonky numbers in the beginning of my data collection and wants to default to the max recorded value of 84Kwh. Or should I seek to delete those weird values from the database. And if so, any suggestions as to where to look to learn about how to do that?

![image](https://github.com/teslamate-org/teslamate/assets/55103311/85c4e69e-e025-49b6-965a-03f65b3e96a1)

**Comment by Regaladfr** [comment]
Same question please. hHow can we set the default battery KWh and % ? Numbers are not saving on the dedicated field.

**Comment by wdywdy6688** [comment]
all dashboards come with teslamate are read only, you need to save a copy to be able to modify/save any parameters.

**Comment by h1ghland3r** [comment]
Is there any way to update the SQL using another application? This way we could have the right information always.
